### DISCUSSION TITLE
options_file structure

### DISCUSSION THREAD

**Discussion by CROdominik** [ERGO-Code/HiGHS#952] [Q&A] [answered]
Hi to everyone, 

I tried to figure that out on multiple sites where some instructions for highs were written but I didn't find anything. 
I'm wondering how exactly should options_file structure look like if I run highs from the command prompt (mac)? Is that txt file, json, something else? I tried several different configurations but I would always get an error message saying that there is an error on line 1. 

Can someone provide an example of the exact file where any 2-3 options are selected/chosen?

Thanks a lot!

**Comment by mckib2** [comment, marked as answer]
Hi @CROdominik, an options file looks like lines of `[thing] = [thing]`, e.g.,

```
presolve = off
solver = ipm
```

If I saved the above in a file named `opt`, I could invoke the HiGHS app with some MPS file like this:

```
./highs --options_file opt my_cool_problem.mps
```
